dace/codegen/targets/sve/unparse.py

```diff
@@ -0,0 +1,562 @@
+                                              stream_type.bytes))
+
+        # Count the number of "to push" elements based on the current predicate
+        self.fill('size_t __cnt = svcntp_b{}({}, {});'.format(
+            self.pred_bits, self.pred_name, self.pred_name))
+
+        # Store the contents of the SVE register in the temporary array
+        self.fill(
+            f'svst1(svwhilelt_b{self.pred_bits}(0, ({self.counter_type}) __cnt), __tmp, '
+        )
+
+        # The contents should be compacted (i.e. all elements where the predicate is true are aligned)
+        self.write(f'svcompact({self.pred_name}, ')
+        self.dispatch_expect(t.value, dtypes.vector(stream_type, -1))
+        self.write('));')
+
+        ptr_cast = ''
+        # Special casting for int64_t back to `long long`
+        if stream_type.type == np.int64:
+            ptr_cast = '(long long*) '
+        elif stream_type.type == np.uint64:
+            ptr_cast = '(unsigned long long*) '
+
+        # Push the temporary array onto the stream using DaCe's push
+        self.fill(f'{target_stream[0]}.push(&__tmp[0], {ptr_cast}__cnt);')
+        self.leave()
+
+    def _Assign(self, t):
+        if len(t.targets) > 1:
+            raise util.NotSupportedError('Tuple output not supported')
+
+        target = t.targets[0]
+
+        if isinstance(target, ast.Name) and target.id in self.stream_associations:
+            # Assigning to a stream variable is equivalent to a push
+            self.push_to_stream(t, target)
+            return
+
+        lhs_type, rhs_type = self.infer(target, t.value)
+
+        if rhs_type is None:
+            raise NotImplementedError(f'Can not infer RHS of assignment ({astunparse.unparse(t.value)})')
+
+        is_new_variable = False
+
+        if lhs_type is None:
+            # The LHS could involve a variable name that was not declared (which is why inference fails)
+            if not isinstance(
+                    target,
+                    ast.Name) or target.id in self.get_defined_symbols():
+                # Either we don't assign to a name, or the variable name has already been declared (but infer still fails, i.e. something went wrong!)
+                raise NotImplementedError('Can not infer LHS of assignment')
+
+            # Declare it as `type name`
+            lhs_type = rhs_type
+            if isinstance(rhs_type, dtypes.vector):
+                # SVE register is possible (declare it as svXXX_t)
+                self.fill(util.TYPE_TO_SVE[rhs_type.type])
+                self.write(' ')
+                # Define the new symbol as vector
+                self.defined_symbols.update({target.id: rhs_type})
+            elif isinstance(rhs_type, dtypes.pointer):
+                raise util.NotSupportedError(
+                    'Defining pointers in Tasklet code not supported')
+
+            # Otherwise, the fallback will grab the case of a scalar, because the RHS is scalar, and the LHS is the same
```

Comment:
```suggestion
            # Otherwise, the fallback will grab the case of a scalar,
            # because the RHS is scalar, and the LHS is the same
```